Makefile

```diff
@@ -77,7 +78,22 @@ clean:
 	rm -rf translate/dist .server-build
 
 shell:
-	"${DC}" run --rm server //bin/bash
+	@container=$$(${DOCKER} ps -q --filter ancestor=local/pontoon | head -n 1); \
+	if [ -z "$$container" ]; then \
+  		echo "Error: No container found based on local/pontoon" >&2; \
+  		exit 1; \
```

Comment:
That means one needs to `make run` before `make shell`, which shouldn't be a requirement.

Reply:
Actually, that doesn't work 🤔

Reply:
> What's the reason for running `make shell` now? I feel like nobody is using it, so maybe assuming that the container is already running is not excessive.

I use it **a lot**. Mostly to then also run `./make shell` (but not always).

Reply:
But that container is currently spun, then removed. I'm guessing you're running commands against the DB, or just checking other things, not interested in preserving those changes in the filesystem?

Reply:
Correct.

We could either use a separate command (e.g. `make shell-server`) to connect to the existing container or fallback to the current behavior (i.e. spin up a new container) if the container is not running.

Reply:
> We could either use a separate command (e.g. `make shell-server`) to connect to the existing container or fallback to the current behavior (i.e. spin up a new container) if the server is not running.

Isn't that what the current code is doing? It only fails if there is no container to spin up.

Reply:
Yup, I didn't realize you already updated the code.